gulpfile.js

```diff
@@ -42,8 +43,17 @@ let config = {
 
 gulp.task( "default", [ "build" ] );
 
+gulp.task( "version", () => {
+	let pk = JSON.parse( fs.readFileSync( "./package.json" ) );
+	return gulp.src( config.source.main )
+		.pipe( replace( /"(\d+\.\d+\.\d+.*)(";)/g, `"${pk.version}";` ) )
```

Comment:
The version follows the semver convention, meaning that it can take forms like `0.35.0-rc.1`. We should find a more elegant way of getting the version from `package.json`